### PR TITLE
LLM: Align converting GPTQ model API with transformer style

### PR DESCRIPTION
**Pull request by leonardozcm**
## Description

Align converting GPTQ model API with transformer style

In this PR we assume users put gptq model and tokenizer.model under the same directory just like huggingface transformer does. 
Usage:
```bash
# tokenizer and model are under the same directory
llm-convert ./ --model-format gptq --model-family llama --outfile out.bin

# Specify tokenizer path manually
llm-convert ./ --model-format gptq --model-family llama --outfile out.bin --tokenizer-path /path/to/tokenizer.model

```

### CODE REVIEW

**Review by Oscilloscope98 — APPROVED** [review]
LGTM

File: python/llm/README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Here is an example to use `llm-convert` command line tool.
 # pth model
 llm-convert "/path/to/llama-7b-hf/" --model-format pth --outfile "/path/to/llama-7b-int4/" --model-family "llama"
 # gptq model
```
Comment (qiuxin2012) [inline python/llm/README.md:41]: Add link https://huggingface.co/TheBloke/vicuna-13B-1.1-GPTQ-4bit-128g/tree/main  here?

Comment (leonardozcm) [reply python/llm/README.md:41]: @Oscilloscope98 pls help me this?

Comment (Oscilloscope98) [reply python/llm/README.md:41]: done :)

**Pull request merged**